pkg/controller/nodenetworkconfigurationpolicy/policyconditions/conditions.go

```diff
@@ -105,31 +105,40 @@ func Update(cli client.Client, policyKey types.NamespacedName) error {
 		}
 
 		nodes := corev1.NodeList{}
-		nodeSelectorFilter := client.MatchingLabels(policy.Spec.NodeSelector)
-		err = cli.List(context.TODO(), &nodes, nodeSelectorFilter)
+		err = cli.List(context.TODO(), &nodes)
 		if err != nil {
 			return errors.Wrap(err, "getting nodes failed")
 		}
-		numberOfNodesAffected := len(nodes.Items)
+		readyNodes := []corev1.Node{}
```

Comment:
We don't need to keep list of nodes, it is not used anywhere. Would you mind just initializing numberOfReadyNodes here and then increment it in the following cycle?

Also, keeping the function that counts ready nodes in a separate function would improve readability.